ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -438,6 +438,9 @@ message DamlContractKeyState {
   // The contract to which the key points to.
   // If unset the key is inactive.
   string contract_id = 1;
+
+  // The time from which the contract is active.
```

Comment:
Please note here how this timestamp is determined (i.e., that it is ledger effective time) or rename it to `active_at_ledger_effective_time` or similar.

Reply:
I used the same name and comment as in `DamlContractState`. I can change it, but then these two fields are inconcistent.

Reply:
Well, if this is a general concept (i.e., fields of type `Timestamp` general refer to a ledger effective time) then this does not need to be changed but we need to make sure we state such context in `daml_kvutils.proto`.

Reply:
I added more comments to both fields in 14a595f.